index.js

```diff
@@ -1,17 +1,151 @@
+    info: {
+      counterpic: './images/Counter.svg',
+      year: 2015,
+      canopy: 'CANOPY',
+      dev: 'Back End Dev',
+    },
+    badges: ['html', 'css', 'bootstap', 'Ruby'],
+    live: 'https://joffreynk.github.io/portfolio-desktop/',
+    github: 'https://github.com/JoffreyNK/portfolio-desktop',
+  },
+  {
+    title: 'Multi-Post Stories',
+    desktop: './images/SnapshootPortfolioD.svg',
+    mobile: './images/tonic2.svg',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text. A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    info: {
+      counterpic: './images/Counter.svg',
+      year: 2015,
+      canopy: 'CANOPY',
+      dev: 'Back End Dev',
+    },
+    badges: ['html', 'css', 'bootstap', 'Ruby'],
+    live: 'https://joffreynk.github.io/portfolio-desktop/',
+    github: 'https://github.com/JoffreyNK/portfolio-desktop',
+  },
+];
 
 if (ul.classList.contains('show')) {
   ul.classList.remove('show');
 }
 
+function clicked() {
+  const shown = document.querySelectorAll('.showpop');
+  shown.forEach((sec) => {
+    sec.classList.remove('showpop');
+  });
+}
+
 links.forEach((link) => {
   link.addEventListener('click', () => {
     ul.classList.remove('show');
+    clicked();
   });
 });
 
 open.addEventListener('click', () => {
   ul.classList.add('show');
 });
+
+const bttns = document.querySelectorAll('.relatepop');
+
+function loop(par) {
+  let text = '';
+  par.forEach((badge) => {
+    text += `<li class='badge'>${badge} </li>`;
+  });
+  return text;
+}
+
+function create(project) {
+  const code = `<ul class='popup'>
+  <li class='popheader'>
+    <h2 class='project-title'>${project.title}</h2>
+    <img class='closeModal' id='closeModel' onclick='clicked();'  src='./images/closeIconpopup.svg' alt='close Icon'>
+  </li>
+  <li class='project-info'>
+    <span class='project-info-items canopy'>${project.info.canopy}</span>
+    <img src=${project.info.counterpic} alt='counter'>
+    <span class='project-info-items dsk-fw'>${project.info.dev}</span>
+    <img src=${project.info.counterpic} alt='counter'>
+    <span class='project-info-items'>${project.info.year}</span>
+  </li>
+  <li class='p-img-container'>
+    <img src=${project.mobile} alt='Tonic Activity on mobile version' class='mobile-image'>
+    <img src=${project.desktop} alt='Tonic activity on desktop version' 
```

Comment:
### OPTIONAL

- [ ] Kindly reduce the height of desktop popup images so that the all contents of the popup can be seen without scrolling 👍🏿 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_208](https://user-images.githubusercontent.com/69638013/169278888-647ac41b-c0d4-4d18-ac58-72463929ec4b.png)  |  ![Screenshot_209](https://user-images.githubusercontent.com/69638013/169279121-d31007f6-594c-4aaf-8dab-18c25280e10d.png)